docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -68,6 +68,16 @@ Malware protection also allows you to manage a blocklist to prevent specified ap
 extending the list of processes that {elastic-defend} considers malicious. Use the **Blocklist enabled** toggle
 to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
 
+When *Detect* malware is enabled, {agent} will quarantine any malicious file it finds. Specifically {agent} will remove the file from its current location, obfuscate it (XOR encrypt it with the encryption key `ELASTIC`), move it to a different folder, and rename it as a string like `318e70c2-af9b-4c3a-939d-11410b9a112c`.
```

Comment:
A few different edits here, to streamline extra phrasing and reduce jargon (I don't know what XOR encryption is, but is mentioning it relevant here?):
```suggestion
When *Detect* is enabled for malware protection, {agent} will quarantine any malicious file it finds. Specifically {agent} will remove the file from its current location, encrypt it with the encryption key `ELASTIC`, move it to a different folder, and rename it as an ID string, such as `318e70c2-af9b-4c3a-939d-11410b9a112c`.
```